build.gradle

```diff
@@ -4,17 +4,13 @@ plugins {
     id 'pmd'
     id 'checkstyle'
     id 'jacoco'
-    id 'war'
+    id 'application'
 }
 
 group 'org.radarcns'
-version '1.0-SNAPSHOT'
+version '0.1.0-SNAPSHOT'
 
-war {
-    baseName = 'redcap'
-    version = project.version
-    extension = 'war'
-}
+mainClassName = 'org.radarcns.redcap.webapp.GrizzlyServer'
 
 targetCompatibility = '1.8'
 sourceCompatibility = '1.8'
```

Comment:
For a standalone distribution like this, we could move to Java 10. There are now docker images for openjdk-10 as well. I'll make a separate issue.

Reply:
Agreed. I will migrate with the next PR of improving exception handling.